packages/neuron-wallet/src/models/asset-account-info.ts

```diff
@@ -0,0 +1,90 @@
+import CellDep, { DepType } from "./chain/cell-dep"
+import Script, { ScriptHashType } from "./chain/script"
+import OutPoint from "./chain/out-point"
+import NetworksService from "services/networks"
+
+interface ScriptCellInfo {
+  cellDep: CellDep
+  codeHash: string
+  hashType: ScriptHashType
+}
+
+export default class AssetAccountInfo {
+  // <genesisBlockHash, sudtScriptCellInfo>
+  private sudtInfo: Map<string, ScriptCellInfo>
```

Comment:
Now we can use private fields of JavaScript

Reply:
I see, but it's not required.